packages/core/http/core-http-server/src/versioning/types.ts

```diff
@@ -251,7 +251,7 @@ export interface AddVersionOpts<P, Q, B> {
    * Validation for this version of a route
    * @experimental
    */
-  validate: false | FullValidationConfig<P, Q, B>;
+  validate: false | FullValidationConfig<P, Q, B> | (() => FullValidationConfig<P, Q, B>); // Provide a way to lazily load validation schemas
```

Comment:
Very naive, perhaps the lazily loaded schemas should be cached? But good enough for PoC.